amazon_msk/metadata.csv

```diff
@@ -100,6 +100,7 @@ aws.msk.kafka.network.RequestMetrics.98thPercentile,gauge,,,,"Attribute exposed
 aws.msk.kafka.network.RequestMetrics.999thPercentile,gauge,,,,"Attribute exposed for management (kafka.network<type=RequestMetrics, name=ResponseQueueTimeMs, request=ListGroups><>999thPercentile)",0,amazon_kafka,,
 aws.msk.kafka.network.RequestMetrics.99thPercentile,gauge,,,,"Attribute exposed for management (kafka.network<type=RequestMetrics, name=ResponseQueueTimeMs, request=ListGroups><>99thPercentile)",0,amazon_kafka,,
 aws.msk.kafka.network.RequestMetrics.Count,gauge,,,,"Attribute exposed for management (kafka.network<type=RequestMetrics, name=ResponseQueueTimeMs, request=ListGroups><>Count)",0,amazon_kafka,,
+aws.msk.kafka.network.RequestMetrics.Count.count,count,,,,"Attribute exposed for management (kafka.network<type=RequestMetrics, name=ResponseQueueTimeMs, request=ListGroups><>Count)",0,amazon_kafka,,
```

Comment:
For these new `*.count` metrics, could we update the descriptions with something ending with `... as a count` to differentiate them from the original metrics?

Reply:
Will do! I couldn't decide if we should put this note into these descriptions or into the gauge versions. Thanks for helping me make a decision!